app/models/author.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from typing import List
+from ..db import db
+
+class Author(db.Model):
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    name: Mapped[str]
+    books: Mapped[List["Book"]] = relationship(back_populates="author")
```

Comment:
I'm still very sad that pylance complains about these forward references. 😭

Reply:
It particularly bugs me because the name is wrapped in quotes as a string, I don't know why Pylance is picking it up 😢

Reply:
OMG. I got it, but it's ugly (though it is the official way to fix the [forward reference](https://peps.python.org/pep-0484/#forward-references)).

author.py needs to import the book module (but not the symbol)
`from . import book`

Then the type would be 
`Mapped[List["book.Book"]]`

Reply:
Wait, wait. It does clear up the pylance error. But then the code doesn't work. 😭

Reply:
An actual solution

in `book.py`
```py
from typing import TYPE_CHECKING
if TYPE_CHECKING:
  from .author import Author
```

In `author.py`
```py
from typing import TYPE_CHECKING
if TYPE_CHECKING:
  from .book import Book
```

Fomr a sqlalchemy github discussion: https://github.com/sqlalchemy/sqlalchemy/discussions/9522